Cargo.toml

```diff
@@ -42,6 +42,7 @@ base64 = "0.13.1"
 bitvec = "1.0.1"
 blockifier = { git = "https://github.com/starkware-libs/blockifier", rev = "ebf48c445b965c5c55b8c724338be7c6e21ffdd4" }
 bytes = "1.4.0"
+axum = { version = "0.6.19", features = ["macros"] }
```

Comment:
cargo-sort should complain

Reply:
And yet it doesn't 🤔 Weird.. thanks manual cargo-sorter.

Reply:
I think cargo-sort is unaware of the workspace toml.